geoips/commandline/args.py

```diff
@@ -560,6 +560,13 @@ def add_args(parser, arglist=None):
             type=str.upper,
         )
 
+    if arglist is None or "no_sectoring" in arglist:
+        procflow_group.add_argument(
+            "--no_sectoring",
+            action="store_true",
+            help="""Specify whether or not you want sectoring to occur.""",
```

Comment:
Maybe this could be worded better. It's not clear what "sectoring" is here.

Something like: `"If true, do not pre-sector data prior to running the algorithm. This is less efficient, but allows raw, unprocessed data to go to the algorithm."`

Reply:
Done.